src/applications/disability-benefits/996/components/VeteranInformation.jsx

```diff
@@ -2,9 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import moment from 'moment';
-import Telephone, {
-  CONTACTS,
-} from '@department-of-veterans-affairs/component-library/Telephone';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/Telephone';
```

Comment:
This will eventually need to be changed to:

```js
import { CONTACTS } from '@department-of-veterans-affairs/component-library';
```

But for now, it needs to remain importing `CONTACTS` from the `Telephone` React component for unit tests to pass - see https://dsva.slack.com/archives/CBU0KDSB1/p1645202969999719